### ISSUE TITLE
Prevent Gridsome from creating certain tags inside <head>

### ISSUE THREAD

**Issue by robinscholz** [gridsome/gridsome#675]
## Description

When working with a custom `index.html` file, Gridsome adds a second `<title>` even though the title hasn’t been specified in any component or the `main.js` file and is already present within the `index.html` file.

### Steps to reproduce

1. Create a custom index.html file with a `<title>` inside `<head>`
2. Run _gridsome build_

### Expected result

It should be possible to control which tags get added to the head of a build.

### Actual result

Gridsome assumes certain fields haven’t been set manually and adds them, even though no information on how to populate them has been specified. The title is just one example of a tag being added although a version of it already exists within the `index.html`

### Environment

<pre><code>
Libs:
- gridsome version: 0.7.3
- @gridsome/cli version: 0.2.3

**Comment by Al-Rozhkov** [comment]
The correct way to populate tags inside `<head>` [described in docs](https://gridsome.org/docs/head/). And I don't think this is going to be changed.

**Comment by robinscholz** [comment]
@Al-Rozhkov The method described in the docs might not always be a viable options for certain projects. I opened this issue because I used gridsome to populate a given template that already included certain head tags. 

In my case gridsome added tags that were already populated, due to some hardcoded values in the gridsome core. It'd be great if there was an option to decide which tags exactly are set by gridsome and which should be omitted.
